README.md

```diff
@@ -266,6 +266,32 @@ extern "Rust" {
     /// `ptr` has to point to the beginning of an allocated block.
     fn miri_static_root(ptr: *const u8);
 
+    /// Miri-provided extern function to obtain a backtrace of the current call stack.
+    /// This returns a boxed slice of pointers - each pointer is an opaque value
+    /// that is only useful when passed to `miri_resolve_frame`
+    fn miri_get_backtrace() -> Box<[*mut ()]>;
```

Comment:
Should we also reserve a flags field here?

Reply:
Done